packages/eas-cli/src/commands/env/exec.ts

```diff
@@ -36,11 +38,16 @@ export default class EnvExec extends EasCommand {
   };
 
   static override flags = {
-    ...EASEnvironmentFlag,
     ...EASNonInteractiveFlag,
   };
 
   static override args = [
+    {
+      name: 'environment',
+      description:
+        "Environment to execute the command in. One of 'production', 'preview', or 'development'.",
+      required: false,
+    },
     {
       name: 'BASH_COMMAND',
       required: true,
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: lines +46 to +54]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->Does oclif recognize that if only 1 string is specified it's bash_command?<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/expo/eas-cli/2654?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->